### DISCUSSION TITLE
AppleTV Notifications

### DISCUSSION THREAD

**Discussion by derflr** [dgreif/ring#809] [Q&A] [answered]
You did a great job puting this together, it's easy to use. The notification popups up on the AppleTV but also includes a doorbell ring, Is there a way to just have the popup and NOT the doorbell ring? I'm referring to the ISO 14 configuration setting, thanks for the help.

**Comment by dgreif** [comment, marked as answer]
Do you have the `sendCameraMotionNotificationsToTv` or `sendDoorbellMotionNotificationsToTv` setting turned on?  If so, showing the notification as a doorbell ring is the _intention_ of the setting. That's the only way to get the notification to automatically stream on the Apple TV.

**Reply by derflr** [reply]
That’s right it does work when I turn that setting on, but it also rings the doorbell. Maybe I can come up with a way to mute the ring sound?

**Reply by dgreif** [reply]
Not sure.  This setting is definitely a hack, and something you have to use at your discretion.  Nothing else I can do to make it work better, Apple has full control over how those notifications are show.

**Comment by derflr** [comment]
Either way, I’m happy to have it, I just sent you a coffee.  Thanks for the coding
